Source/ElasticLINQ/ElasticLINQ.csproj

```diff
@@ -1,166 +1,13 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{875E1599-5B7F-4084-BBE0-0D70C3C80428}</ProjectGuid>
```

Comment:
How are you converting these csproj files?  I'm not keen on losing the ProjectGuid as it will mess up people upgrading, strong name signing, internals visible to...

Reply:
So originally, I actually just deleted the csproj and used the template on MSDN, I'm okay with adding back in the project guids if you'd like.